firmware-src/pages/pwm.html

```diff
@@ -2,31 +2,26 @@
 <head>
 	<link rel="stylesheet" type="text/css" href="style.css">
 	<title>PWM</title>
-	<script type="text/javascript" src="librest.js"></script>
-	<script type="text/javascript">
-		function print(text, err = false) {
-			document.getElementById('output').style.color = err ? 'red' : 'green';
-			document.getElementById('output').innerHTML = text;
-		}
-
-		function send(dc, count, frequency) {
+	<script src="libs.js"></script>
+	<script>
+		function setPwm(dc, count, frequency) {
 			print('');
-			var params = {}
-			params[document.getElementById('pin').value] = dc;
+			var params = {};
+			params[byId('pin').value] = dc;
 			params['frequency'] = frequency;
 			params['count'] = count;
-			rest_common('pwm/control', params, function (err, res) {
+			send_command('pwm/control', params, function (err, res) {
```

Comment:
Uncaught DOMException: Failed to execute 'setRequestHeader' on 'XMLHttpRequest': 'Bearer function (err, res) {
				if(err)
					print('ERROR: ' + res, true);
			}' is not a valid HTTP header field value.
    at ajax (http://esp-test.vrn.dataart.net/libs.js:5:10)
    at send_command (http://esp-test.vrn.dataart.net/libs.js:22:2)
    at setPwm (http://esp-test.vrn.dataart.net/pwm.html:13:4)
    at HTMLButtonElement.onclick (http://esp-test.vrn.dataart.net/pwm.html:45:85)